tests/Bridge/Doctrine/Persister/ObjectManagerPersisterTest.php

```diff
@@ -183,7 +204,8 @@ public function provideEntities()
                 $dummy->id = 300;
 
                 return $dummy;
-            })()
+            })(),
+            true
```

Comment:
I have re-added here because otherwise that test does not make sense there. Not sure why it was removed because it was present in the original PR: https://github.com/theofidry/AliceDataFixtures/pull/68